quickquickstarts/Using_Validators_with_a_Linux_image/validators_with_a_linux_image.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Can you please share details on how this has been tested?
For example, a customer would likely copy-paste this text; look for everything in angle brackets "<>" and replace with custom values. At that point the template should just work.
Has such a run been done on this quickstart example?